### DISCUSSION TITLE
Could not find org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.21.

### DISCUSSION THREAD

**Discussion by Tazkerah** [ionic-team/capacitor#3334] [Q&A]
new capacitor project (contain only www) when open it in android studio i can't sync the gradle 
(Could not find org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.21.
Searched in the following locations:
  - file:/C:/Users/i.tech/.android/manual-offline-m2/android-gradle-plugin-3.5.0-beta01/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.pom
  - file:/C:/Users/i.tech/.android/manual-offline-m2/android-gradle-plugin-3.5.0-beta01/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.jar
  - file:/C:/Users/i.tech/.android/manual-offline-m2/gmaven_stable/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.pom
  - file:/C:/Users/i.tech/.android/manual-offline-m2/gmaven_stable/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.jar
  - https://dl.bintray.com/kotlin/kotlin-eap/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.pom
  - https://dl.bintray.com/kotlin/kotlin-eap/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.jar
  - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.pom
  - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.jar
Required by:
    project : > com.android.tools.build:gradle:3.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.analytics-library:shared:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.lint:lint-gradle-api:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:gradle-api:3.4.0
    project : > com.android.tools.build:gradle:3.4.0 > androidx.databinding:databinding-compiler-common:3.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0 > com.android.tools:sdk-common:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0 > com.android.tools:common:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0 > com.android.tools.build:manifest-merger:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0 > com.android.tools.analytics-library:tracker:26.4.0
    project : > com.android.tools.build:gradle:3.4.0 > com.android.tools.build:builder:3.4.0 > com.android.tools:sdklib:26.4.0 > com.android.tools:repository:26.4.0
) 
I am not using any kotlin in my project and i don't have it one the machine. any idea why this is happening?

**Comment by carlpoole** [comment]
Hi there, what version of Android Studio are you using?

**Comment by Tazkerah** [comment]
4.0.1

**Comment by Tazkerah** [comment]
my internet connection is really slow, so i am trying to build offline, I've downloaded the offline components (the gradle and the maven) with the gradle initial script but this error baffles me, as i don't use kotlin, and these files don't seem to be exist on the suggested pathes (as https://dl.bintray.com/kotlin/kotlin-eap/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.21/kotlin-stdlib-jdk8-1.3.21.pom as I get "The requested path was not found." on my browser)

**Comment by jcesarmobile** [comment]
Capacitor uses `com.android.tools.build:gradle:3.6.1`, but your computer is trying to download `3.4.0`, so something is wrong.

Can you provide a sample app?

**Comment by Tazkerah** [comment]
update: I'm using gradle 6.5 and gradle plugin 3.5.0beta02 because this was version of the offline maven repository, the old error has resolved but now I'm getting new error
 Could not find gradle-3.6.1.jar (com.android.tools.build:gradle:3.6.1).
Searched in the following locations:
    https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/3.6.1/gradle-3.6.1.jar

how to change capacitor gradle version or
how can make capacitor search in the local repository
